functions/transformJson.js

```diff
@@ -1,4 +1,4 @@
-const addArray = datas => (
+const transformJson = datas => (
```

Comment:
Which Json ? datas from which entities ?

Reply:
i have change filename to transformPoiSampleJson